scripts/zephyr_module.py

```diff
@@ -375,6 +375,16 @@ def main():
 
     if args.settings_out:
         with open(args.settings_out, 'w', encoding="utf-8") as fp:
+            fp.write('''\
+# WARNING. THIS FILE IS AUTO-GENERATED. DO NOT MODIFY!
```

Comment:
I believe that all files in a build folder should be considered generated :stuck_out_tongue: 
but I see no problem in having the warning though.

Reply:
> I believe that all files in a build folder should be considered generated stuck_out_tongue

Yeah, but other files in the build directory contain similar warnings. The "DO NOT MODIFY" part is also important IMO. Thanks for review.

Reply:
> > I believe that all files in a build folder should be considered generated stuck_out_tongue
> 
> Yeah, but other files in the build directory contain similar warnings. The "DO NOT MODIFY" part is also important IMO. Thanks for review.

It was there:
https://github.com/zephyrproject-rtos/zephyr/pull/13067#discussion_r254249203

It was gone:
https://github.com/zephyrproject-rtos/zephyr/pull/14667#discussion_r267288049

and now it's back.

Just waiting for the next person to decide it can be removed again :laughing: